### DISCUSSION TITLE
Device availability?

### DISCUSSION THREAD

**Discussion by bert269** [toverainc/willow#50] [General]
Any idea on where to get this hardware?
Has it even been produced yet, or is it still in planning stages?

**Comment by kristiankielhofner** [comment]
Hi!

Espressif has been making and selling the ESP BOX for a year or so.

However, their distributors never kept much stock on hand because until Willow there wasn't any software for them and they weren't selling. With the announcement of Willow on Monday distributors sold out worldwide within 36 hours or so.

I know how frustrating it can be to not be able to get hardware but we know distributors have ordered more from Espressif and depending on lead time, etc we hope distributors have ample supply within a month or so (the slow boat from China, sadly).

**Reply by winghavenrc** [reply]
I ignored your advice (and will regret it I'm sure) and ordered what might have been the last ESP Box Lite from Adafruit... wanted to get started playing...

**Reply by kristiankielhofner** [reply]
We weren't planning on officially supporting the Lite but many people (in desperation) bought them anyway so the team and I would like to do our very best to not leave them out in the cold.

The Lite has full functionality with Willow today other than the front buttons because we don't have much of a UI and I don't know what we would do with them (yet).

Just make sure to complete this one extra [step](https://github.com/toverainc/willow#esp-box-lite-note-for-users-who-purchased-the-esp-box-lite) and you should be good-to-go. I periodically flash on the Lites I have as do some of the other developers.

**Comment by NobleWolf** [comment]
> Any idea on where to get this hardware? Has it even been produced yet, or is it still in planning stages?

I'll sell mine for $100USD 😂
